runtime/nvqir/cutensornet/tensornet_state.cpp

```diff
@@ -579,8 +550,15 @@ TensorNetState::createFromStateVector(std::span<std::complex<double>> stateVec,
   // previous state should be in the tensor network form. Construct the state
   // projector matrix
   // FIXME: use CUDA toolkit, e.g., cuBlas, to construct this projector matrix.
-  auto ket =
-      Eigen::Map<const Eigen::VectorXcd>(stateVec.data(), stateVec.size());
+  Eigen::VectorXcd ket(stateVec.size());
+  for (std::size_t i = 0; i < stateVec.size(); ++i) {
+    std::bitset<64> bs(i);
+    std::string bitStr = bs.to_string();
+    std::reverse(bitStr.begin(), bitStr.end());
+    bitStr = bitStr.substr(0, numQubits);
+    ket[std::stoull(bitStr, nullptr, 2)] = stateVec[i];
```

Comment:
Was this same code up above?

Reply:
Yes, it's a duplicate. Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1630/commits/97532463b0e9d698433ae2b018c818fca8681bcc by creating a common helper.